virtinst/install/urldetect.py

```diff
@@ -46,6 +46,9 @@ def acquire_file_content(self, path):
             self._filecache[path] = content
         return self._filecache[path]
 
+    def paths_exist(self, paths):
+        return all([self._fetcher.hasFile(p) for p in paths])
+
```

Comment:
this seems unused?

Reply:
oh yes, thanks for noticing!
let me remove this and give it a re-test once